src/types/bibDetail.tsx

```diff
@@ -18,3 +23,8 @@ export interface Url {
   url: string
   urlLabel: string
 }
+
+export interface FieldMapping {
```

Comment:
This is already defined in bibTypes.

Reply:
yea these are just some artifacts of the two pr's being separate. I merged them into one bibTypes file.